lib/measure_repository_service_test_kit/library_package.rb

```diff
@@ -132,13 +134,32 @@ class LibraryPackage < Inferno::TestGroup
       uses_request :library_package
 
       run do
-        assert(related_artifacts_present?(resource))
+        assert(related_artifacts_present?(resource, false))
       end
     end
 
     test do
-      title 'Throws 404 when no Library on server matches id'
+      optional
```

Comment:
Nitpick: Maybe optional tests should be the last ones in a group? Not sure, but that feels logical to me rather than having an optional test sandwiched between a bunch of other required ones. Thoughts?